templates/partial/_navigation-careers.html

```diff
@@ -0,0 +1,96 @@
+          <p>You love to make software work perfectly and efficiently. Your analytical approach, your intellect, your teamwork and your work ethic put you at the top of your game. You are self-motivated and organised. And you really care to work on a platform that is freely available to everybody on the planet.</p>
+          <p><a href="/careers/engineering">More&hellip;</a></p>
+          <div class="u-hide u-show--small">
+            <small>Europe | Americas | Asia-Pacific | Middle East | Africa</small>
+            <p><a href="/careers/engineering#apply">Apply now&nbsp;&rsaquo;</a></p>
+            <hr />
+          </div>
+        </div>
+        <div class="col-3">
+          <h4 class="p-heading--four is-dense">Marketing</h4>
+          <p>We deliver the right technology message to a highly competent audience, at exactly the right moment. Success depends on having good instincts for tech trends and topics and a natural inclination to leadership.</p>
+          <p>The best candidates for marketing at Canonical are tech-savvy and are comfortable working with both data and messaging.</p>
+          <p>You’re excited to deliver measurable results. You turn to data for insights &mdash; in spreadsheets, business analytics suites and databases. You love data visualisation and presentations based on fact.</p>
+          <p>You are also passionate about message and language, and strive to find the best way to share a powerful idea that drives action.</p>
+          <p><a href="/careers/marketing">More&hellip;</a></p>
+          <div class="u-hide u-show--small">
+            <small>Europe | Americas</small>
+            <p><a href="/careers/marketing#apply">Apply now&nbsp;&rsaquo;</a></p>
+            <hr />
+          </div>
+        </div>
+        <div class="col-3">
+          <h4 class="p-heading--four is-dense">Sales</h4>
+          <p>You love enterprise technology and you love doing business. You enjoy helping customers embrace newer and better ways to deliver services, solutions and systems.</p>
+          <p>Your approach is consultative, you are results and goal oriented, and you know how to invest your time. Most importantly, you care that the customer gets the best result &mdash; every time. You know that long term relationships require commitment and continuity, patience and precision. That’s what makes you different.</p>
+          <p><a href="/careers/sales">More&hellip;</a></p>
+          <div class="u-hide u-show--small">
+            <small>Europe | Americas | Asia-Pacific | Middle East | Africa</small>
+            <p><a href="/careers/sales#apply">Apply now&nbsp;&rsaquo;</a></p>
+            <hr />
+          </div>
+        </div>
+        <div class="col-3">
+          <h4 class="p-heading--four is-dense p-muted-heading" style="padding-top: .5rem; margin-bottom: 1.5rem;">Explore opportunities in</h4>
+          <ul>
+            <li><a href="/careers/alliances">Alliances&nbsp;&rsaquo;</a></li>
```

Comment:
We don't have `Alliances`, `Business Development`, etc.
Here is a list with our departments:
Engineering
Tech-ops
Commercial-ops
Sales
Marketing
Design
Project Management
Finance
Legal
Admin
HR

Reply:
Updated the links, cheers.